packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -104,6 +154,53 @@ describe('quantic-sort', () => {
           });
       });
 
+      describe('when passing custom options as slots to the component', () => {
+        it('should display the custom options properly', () => {
+          visitSort({useCase: param.useCase}, true, {
+            withCustomSortOptions: true,
+          });
+
+          Expect.displaySortDropdown(true);
+          Expect.labelContains('Sort by');
+
+          Actions.openDropdown();
+
+          Expect.containsOptions(customSortOptionsValues);
+          Expect.optionsEqual(customSortOptions);
+        });
+
+        customSortOptionsValues
+          .filter((value) => value !== defaultCustomOptionValue)
```

Comment:
this should not be needed here, why are adding a default sort option to `customSortOptionsValues ` at the first place?

Reply:
I will remove and change the placeholder to the value getter. This pattern was something you added in the other test with the default sort options because of the placeholder value.

Reply:
this is also the pattern used in the "default" sort options...
This is there because by default, the first item is selected and the value of the dropdown set to that first item. The reason we skip the first one, is that because it is already selected, selecting it again does not trigger a new search request

Reply:
@SimonMilord This isn't resolved?

Reply:
resolved now :P